pkg/trace/stats/concentrator.go

```diff
@@ -185,8 +185,9 @@ func computeStatsForSpanKind(s *pb.Span) bool {
 
 // Input specifies a set of traces originating from a certain payload.
 type Input struct {
-	Traces      []traceutil.ProcessedTrace
-	ContainerID string
+	Traces        []traceutil.ProcessedTrace
+	ContainerID   string
+	ContainerTags string
```

Comment:
As in my comment above, I think we can avoid the overhead of doing a string join and later a split if we just keep the slice as is, do a hash of it (so some extra overhead), and then manage the hash and slice like we do today for peer tags.